docs/statement.md

```diff
@@ -0,0 +1,11 @@
+# `jupyter` accessibility statement
+
+As of Jan 31, 2022, `jupyterlab` nor `retrolab` meet WCAG 2.1 Level A compliance. 
```

Comment:
Maybe worth adding - if you want to keep track of concrete tasks towards making these WCAG compliant check the roadmap and this and that board?